README.md

```diff
@@ -43,20 +46,14 @@ wget https://github.com/dipdup-io/stone-packaging/releases/latest/download/cpu_a
 
 ### Creating and Verifying a Test Proof Using Binaries
 
-Clone the repository:
-
-```bash
-git clone https://github.com/dipdup-io/stone-packaging.git /tmp/stone-packaging
-```
+Note: You still need to clone the repo to create and verify a test proof.
```

Comment:
Let's leave it as it was before

Reply:
ok will do that. implementing changes now